install_config/configuring_kuryrsdn.adoc

```diff
@@ -157,3 +157,9 @@ kuryr-controller-59fc7f478b-qwk4k   1/1       Running   0          3d        192
 
 kuryr-cni pods should run on every {product-title} node. Single
 kuryr-controller instances should run on any of the nodes.
+
+[NOTE]
+====
+When using Kuryr SDN, network policies and nodeport services are not currently
```

Comment:
Suggestion:

Network policies and nodeport services are not supported when Kuryr SDN is enabled.